packages/cactus-plugin-keychain-vault/src/test/typescript/integration/plugin-keychain-vault.test.ts

```diff
@@ -1,14 +1,40 @@
-// import test, { Test } from "tape-promise/tape";
+import test, { Test } from "tape-promise/tape";
+import { v4 as internalIpV4 } from "internal-ip";
 
-// import { v4 as uuidv4 } from "uuid";
-// import {
-//   IPluginKeychainVaultOptions,
-//   PluginKeychainVault,
-// } from "../../../main/typescript/public-api";
+import express from "express";
+import bodyParser from "body-parser";
+import http from "http";
+import { AddressInfo } from "net";
+
+import {
+  Containers,
+  K_DEFAULT_VAULT_DEV_ROOT_TOKEN,
+  K_DEFAULT_VAULT_HTTP_PORT,
+  VaultTestServer,
+} from "@hyperledger/cactus-test-tooling";
+
+import { v4 as uuidv4 } from "uuid";
+
+import {
+  LogLevelDesc,
+  IListenOptions,
+  Servers,
+} from "@hyperledger/cactus-common";
+
+import {
+  IPluginKeychainVaultOptions,
+  PluginKeychainVault,
+} from "../../../main/typescript/public-api";
+
+import { K_CACTUS_KEYCHAIN_VAULT_TOTAL_KEY_COUNT } from "../../../main/typescript/prometheus-exporter/metrics";
+
+import { DefaultApi as KeychainVaultApi } from "../../../main/typescript/public-api";
+
+const logLevel: LogLevelDesc = "TRACE";
 
 // test("PluginKeychainVault", (t1: Test) => {
```

Comment:
@jagpreetsinghsasan If we don't need this code then it's best to remove it. Or if you plan to add it back in the future and left it here commented so that it doesn't have to be written from scratch again (which is a valid reason IMO) then just make sure to leave a TODO item in the code explaining the latter and also open a GH issue for it.
I'm guilty of this too, so apologies for not setting the right example here (I was the one originally commenting out this test without any explanation so shame on me for that... :-) )

Reply:
https://github.com/hyperledger/cactus/issues/676